esp/templates/survey/completed_survey.html

```diff
@@ -10,5 +10,12 @@
 <br />
 <p>Your survey responses for {{ prog.niceName }} have been saved.  We appreciate the help.</p>
 
+{% if "teach" in request.path %}
```

Comment:
This looks like it might be a good change to keep?

Reply:
Sure!